src/components/uielements/backLink/BackLink.tsx

```diff
@@ -1,25 +1,25 @@
 import React from 'react'
 
 import { LeftOutlined } from '@ant-design/icons'
-import { Link } from 'react-router-dom'
+import { useHistory } from 'react-router-dom'
 
 import { BackLinkWrapper } from './BackLink.style'
 
 type Props = {
-  route: string
   title?: string
 }
 
-const BackLink: React.FC<Props> = ({ route, title = 'Back', ...otherProps }): JSX.Element => {
+const BackLink: React.FC<Props> = ({ title = 'Back', ...otherProps }): JSX.Element => {
```

Comment:
agree here. in general IMAO its better not to use spread at all in case you know the props type. its more explicit in render itself. for example
```ts
const Foo: React.FC<{click: () => void}> = ({click} ) => {
  // a lot of code here
  return <button onClick={click}>click me</button>
}
```
in this case i don't know that this click handler is just a proxied value from props and can suppose that this is some 'computed' value calculated somewhere inside of the component's function. And this point is also relevant during reviewing PRs when you dont see whole function's body and have to unwrap whole function's body to understand that it was just a value from props.

more expicit way to use props' values directly

```ts
const Foo: React.FC<{click: () => void}> = (props ) => {
  // a lot of code here
  return <button onClick={props.click}>click me</button>
}
```